### PR TITLE
Merge AAD auth support from feature branch into master

### PR DESCRIPTION
**Pull request by timtay-microsoft**
This won't be merged until it is closer to time to release this feature, as the service itself doesn't support this in every region just yet

### CODE REVIEW

File: vsts/sdl.yaml [hunk 1]
```diff
@@ -20,6 +20,7 @@ phases:
     displayName: CredScan
     inputs:
       debugMode: false
+      suppressionsFile: vsts/CredScanSuppressions.json
```
Comment (timtay-microsoft) [inline vsts/sdl.yaml:23]: This PR brought new samples with it, and one of the samples has a false postive credscan case that this suppressions file accounts for

**Review by barustum — APPROVED** [review]

**Pull request merged**